lite/examples/object_detection/android/README.md

```diff
@@ -14,7 +14,7 @@ explicitly.
 
 Application can run either on device or emulator.
 
-<!-- TODO(b/124116863): Add app screenshot. -->
+<img src="https://www.tensorflow.org/lite/examples/image_classification/images/android_banana.png" width="350" />
```

Comment:
This is image classification not object detection.

Reply:
@lintian06  thank you for reviewing 
I have updated the link to https://www.tensorflow.org/lite/examples/object_detection/images/android_apple_banana.png